### DISCUSSION TITLE
DataTable: add a tooltip to a cell when hovering

### DISCUSSION THREAD

**Discussion by annette-graham** [primefaces/community#2040] [PrimeReact] [answered]
I'm attempting to display a tooltip when a user hovers over a cell (or row) in my data table, but am struggling to get it to work. For added context, the rows are selectable and the cells are editable in my table

Is there an example somewhere that I am able to look at? 

Thanks :)

**Comment by melloware** [comment, marked as answer]
Yep I posted a solution here: https://github.com/primefaces/primereact/issues/5620

**Comment by annette-graham** [comment]
Ah perfect, that's exactly what I'm after. Thank you!
